labels.py

```diff
@@ -0,0 +1,39 @@
+import json
```

Comment:
Also, do we need this?

Reply:
ups.. my testing file :/ removing it in aaaccd9